Dockerfile

```diff
@@ -81,7 +81,7 @@ RUN CHART_VERSION="1.9.604"                   CHART_FILE=/charts/rke2-cilium.yam
 RUN CHART_VERSION="v3.13.300-build2021022306" CHART_FILE=/charts/rke2-canal.yaml          CHART_BOOTSTRAP=true   /charts/build-chart.sh
 RUN CHART_VERSION="1.10.101-build2021022303"  CHART_FILE=/charts/rke2-coredns.yaml        CHART_BOOTSTRAP=true   /charts/build-chart.sh
 RUN CHART_VERSION="1.36.301"                  CHART_FILE=/charts/rke2-ingress-nginx.yaml  CHART_BOOTSTRAP=false  /charts/build-chart.sh
-RUN CHART_VERSION="v1.20.10-rke2r1-build2021081601" \
+RUN CHART_VERSION="v1.20.10-rke2r1-build2021082001" \
```

Comment:
Only the "metadata" moved?

Reply:
Yeah, the dates were wrong in the tags I created. I forgot the build system is in UTC and it was off.